src/scss/components/menu/_index.scss

```diff
@@ -56,13 +56,13 @@ hx-menuitem {
   &:hover {
     background-color: transparent;
     border-color: transparent;
-    color: $cyan-500;
+    color: var(--hxMenu-hxMenuitem-hover-color, $cyan-500);
```

Comment:
Please update color variable to Color Palette 2.0 specs.

Reply:
Changes have been done, Please review